core/services/registrysyncer/local_registry.go

```diff
@@ -0,0 +1,81 @@
+package registrysyncer
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	p2ptypes "github.com/smartcontractkit/chainlink/v2/core/services/p2p/types"
+)
+
+type CapabilityID string
+type DonID uint32
+
+type DON struct {
+	capabilities.DON
+	CapabilityConfigurations map[CapabilityID]capabilities.CapabilityConfiguration
+}
+
+type Capability struct {
```

Comment:
This is `CapabilityInfo` just grabbed from on-chain registry

Reply:
@DeividasK Thinking about this more, I don't think this is entirely correct: the CapabilityInfo contains DON information, whereas this should just contain the Capability information without the DON part (that's actually what this struct does today).

Do you mind if I think about this a bit more? I'm thinking ahead at how this would work if eg. a Capability could exist on multiple DONs -- this is not the case today but is likely to be relevant for scaling efforts in the future.